variantrule/Modnar; Character Points.json

```diff
@@ -0,0 +1,489 @@
+{
+	"_meta": {
+		"sources": [
+			{
+				"json": "Giddy",
+				"abbreviation": "Brew",
+				"full": "Character Point System",
+				"authors": [
+					"Modnar"
+				],
+				"version": "1.0.0",
+				"url": "https://www.gmbinder.com/share/-L8YS8M9xAdPhqY7d6tj",
+				"targetSchema": "1.0.0"
+			}
+		]
+	},
+
+	"type": "section",
+	"name": "Character Points [CP]",
+	"entries": [
```

Comment:
Not quite sure what you're aiming for here -- if this is a single variant rule, it should be stored in an array with key `variantrule`. See other files for examples. Alternatively, I can easily fix it up, if you wish (and tell me which things are supposed to be individual rules).